js/main.js

```diff
@@ -46,57 +40,54 @@ function handleViewNavigation(event) {
   if (!event.target.matches('A')) {
     return;
   }
+  data.editing = null;
   switchView(event.target.getAttribute('data-view'));
+  $formJournal.reset();
+  $imageDisplay.setAttribute('src', 'images/placeholder-image-square.jpg');
 }
 
 function imgUrlInput(event) {
   var imgsrc = event.target.value;
   $imageDisplay.setAttribute('src', imgsrc);
 }
 
-function editImgUrlInput(event) {
-  var imgsrc = event.target.value;
-  $editImageDisplay.setAttribute('src', imgsrc);
-}
-$editPhotoUrl.addEventListener('input', editImgUrlInput);
-
-function editformJournalSubmit(event) {
+function formJournalSubmit(event) {
   event.preventDefault();
-  var editObj = {};
-  editObj.title = $editFormJournal.elements.editTitle.value;
-  editObj.imgsrc = $editFormJournal.elements.editPhotoUrl.value;
-  editObj.notes = $editFormJournal.elements.editNotes.value;
-  editObj.entryId = data.editing;
-  for (var i = 0; i < $liElements.length; i++) {
-    if (parseInt($liElements[i].getAttribute('data-entry-id')) === parseInt(data.editing)) {
-      $liElements[i].replaceWith(domTree(editObj));
+  if (data.editing !== null) {
+    var editObj = {};
+    editObj.title = $formJournal.elements.title.value;
+    editObj.imgsrc = $formJournal.elements.photoUrl.value;
+    editObj.notes = $formJournal.elements.notes.value;
+    editObj.entryId = data.editing;
```

Comment:
You shouldn't need to reassign the `id`. It's the same entry, so no new id value is needed.